tests/test_bootpolicy.robot

```diff
@@ -0,0 +1,187 @@
+    ${valueDict} =   create dictionary   data=${bootpolicy}
+    Write Attribute    /org/openbmc/settings/host0   boot_policy   data=${valueDict}
+    Read the Attribute  /org/openbmc/settings/host0    boot_policy
+    Response Should Be Equal   ONETIME
+    ${output} =    Run IPMI Standard command   chassis bootparam get 5
+    Should Contain   ${output}   Options apply to only next boot
+
+Set Permanent boot order using REST
+    [Documentation]   This testcase is to set permanent boot policy using REST
+    ...               URI and then verify using REST API and ipmitool.\n
+   
+    ${bootpolicy} =   Set Variable   PERMANENT
+    ${valueDict} =   create dictionary   data=${bootpolicy}
+    Write Attribute    /org/openbmc/settings/host0   boot_policy   data=${valueDict}
+    Read the Attribute  /org/openbmc/settings/host0    boot_policy
+    Response Should Be Equal   PERMANENT
+    ${output} =    Run IPMI Standard command   chassis bootparam get 5
+    Should Contain   ${output}   Options apply to all future boots
+
+Set Onetime boot order using IPMITOOL
+    [Documentation]   This testcase is to set boot policy to onetime boot using ipmitool
+    ...               and then verify using REST URI and ipmitool.\n
+
+    Run IPMI command   0x0 0x8 0x05 0x80 0x00 0x00 0x00 0x00
+    Read the Attribute   /org/openbmc/settings/host0   boot_policy
+    Response Should Be Equal   ONETIME
+    ${output} =   Run IPMI Standard command   chassis bootparam get 5
+    Should Contain   ${output}   Options apply to only next boot
+
+   
+Set Permanent boot order using IPMITOOL
+    [Documentation]   This testcase is to set boot policy to permanent using ipmitool
+    ...               and then verify using REST URI and ipmitool.
+
+    Run IPMI command   0x0 0x8 0x05 0xC0 0x00 0x00 0x00 0x00
+    Read the Attribute   /org/openbmc/settings/host0   boot_policy
+    Response Should Be Equal   PERMANENT
+    ${output} =   Run IPMI Standard command   chassis bootparam get 5
+    Should Contain   ${output}   Options apply to all future boots
+
+Boot order with permanent boot policy
+    [Documentation]   This testcase is to verify that boot order does not change
+    ...               after first boot when boot policy set to permanent
+
+    Power Off Host
+
+    ${bootpolicy} =   Set Variable   PERMANENT
+    ${valueDict} =   create dictionary   data=${bootpolicy}
+    Write Attribute    /org/openbmc/settings/host0   boot_policy   data=${valueDict}
+    Read the Attribute  /org/openbmc/settings/host0    boot_policy
+    Response Should Be Equal   PERMANENT
+
+    ${bootDevice} =   Set Variable   CDROM
+    ${valueDict} =   create dictionary   data=${bootDevice}
+    Write Attribute    /org/openbmc/settings/host0   boot_flags   data=${valueDict}
+    Read the Attribute  /org/openbmc/settings/host0    boot_flags
+    Response Should Be Equal   CDROM
+
+    Power On Host
+
+    Read the Attribute  /org/openbmc/settings/host0    boot_policy
```

Comment:
Can we use "Read Attribute" in lib/rest_client.robot instead of "Read the Attribute"?